_drafts/SC1-3-5-autocomplete-valid.md

```diff
@@ -0,0 +1,181 @@
+---
+name: Autocomplete valid
+description: |
+  This rule checks that the HTML autocomplete attribute has a correct value
+
+success_criterion:
+- 1.3.5 # Identify Input Purpose
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Wilco Fiers
+---
+
+## Test Procedure
+
+### Applicability
+
+The rule applies to any HTML `input`, `select` and `textarea` element with a [non-empty]() `autocomplete` attribute, except if one of the following is true:
+
+- The element is **not** [[visible on the page]] or [[exposed to assistive technologies]]
```

Comment:
definition needs to follow convention [text in html file](#unique-id-of-definition)